src/libsync/owncloudpropagator.cpp

```diff
@@ -379,6 +383,12 @@ void OwncloudPropagator::start(const SyncFileItemVector& items)
                 currentDirJob->append(dir);
             }
             directories.push(qMakePair(item->destination() + "/" , dir));
+        } else if (bundledRequestsEnabled
+                  && (item->_instruction == CSYNC_INSTRUCTION_NEW)
```

Comment:
Good that it is only for new files, maybe for updating files there is too many other things to consider for now, e.g. If-Etag-match and permissions and concurrency/locks etc.